doc/api/fs.md

```diff
@@ -3019,7 +3023,21 @@ changes:
                  it will emit a deprecation warning with id DEP0013.
 -->
 
+> Stability: 1 - Recursive removal is experimental.
+
 * `path` {string|Buffer|URL}
+* `options` {Object}
+  * `emfileWait` {integer} If an `EMFILE` error is encountered, Node will retry
```

Comment:
This nit will need rewrapping, unfortunatelly:
```suggestion
  * `emfileWait` {integer} If an `EMFILE` error is encountered, Node.js will retry
```

Reply:
Maybe this instead?

```suggestion
  * `emfileWait` {integer} If an `EMFILE` error is encountered, retry
 ```

Reply:
Er, maybe not. I'll remove my comment (and this one).